sdcm/sla/sla_tests.py

```diff
@@ -123,12 +123,25 @@ def validate_role_service_level_attributes_against_db():
                 role_for_attach.attach_service_level(new_service_level)
                 role_for_attach.validate_role_service_level_attributes_against_db()
 
-                # Workaround for issue https://github.com/scylladb/scylla-enterprise/issues/2572
-                if restart_scylla:
```

Comment:
restart_scylla  after this PR will be unused. so it needs to be deleted from "attach_sl_and_validate_scheduler_runtime" method and it usage

Reply:
removed